buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1394,5 +1420,31 @@ artifactId=${project.name}
         args argsNeeded
       }
     }
+
+
+    /** ***********************************************************************************************/
+
+    // Method to create the PortableValidatesRunnerTask.
+    // The method takes PortableValidatesRunnerConfiguration as parameter.
+    project.ext.createPortableValidatesRunnerTask = {
+      def config = it ? it as PortableValidatesRunnerConfiguration : new PortableValidatesRunnerConfiguration()
+      def name = config.name
+      def beamTestPipelineOptions = [
+        "--runner=org.apache.beam.runners.reference.testing.TestPortableRunner",
+        "--jobServerDriver=${config.jobServerDriver}",
+      ]
+      if(config.jobServerConfig){
+        beamTestPipelineOptions.add("--jobServerConfig=${config.jobServerConfig}")
+      }
+      project.tasks.create(name: name, type: Test) {
+        group = "Verification"
+        description = "Validates the PortableRunner with JobServer ${config.jobServerDriver}"
+        systemProperty "beamTestPipelineOptions", JsonOutput.toJson(beamTestPipelineOptions)
+        classpath = config.testClasspathConfiguration
+        testClassesDirs = project.files(project.project(":beam-sdks-java-core").sourceSets.test.output.classesDirs, project.project(":beam-runners-core-java").sourceSets.test.output.classesDirs)
```

Comment:
Ah, I just realized that it's possible this will fail on submodules that have not already added an evaluation dependency on beam-sdks-java-core (necessary as we're accessing this source set here).

@lukecwik Do you know if there's a way to do this from within the plugin? Is that even recommended if possible? Can we instead hard fail with a message if the evaluation dependency has not been specified?

Reply:
Another question about the structure here: Should we add this within `applyJavaNature`? The `ValidatesRunner` tests only work for Java-based runners anyway (and only the only runners that exist are written in Java).

Reply:
Scratch that last comment. I misread the surrounding context.

Reply:
I suppose the gradle task will fail verbosely if the down stream project does not have dependency on :beam-sdks-java-core 
The best I can think of at the moment is just document expectation for :beam-sdks-java-core dependency.

Reply:
OK, that sounds good for now.

Reply:
you should be able to add:
```
project.evaluationDependsOn(":beam-sdks-java-core")
project.evaluationDependsOn(":beam-runners-core-java")
```

Don't forget to update the comment as well.